src/ring_theory/derivation.lean

```diff
@@ -717,6 +717,10 @@ begin
     derivation.tensor_product_to_tmul, one_smul, D.map_one_eq_zero, smul_zero, sub_zero],
 end
 
+@[simp] lemma derivation.lift_kaehler_differential_comp_D (D : derivation R S M) (x : S) :
```

Comment:
```suggestion
@[simp] lemma derivation.lift_kaehler_differential_comp_D (D' : derivation R S M) (x : S) :
```
It seems a little confusing to me to have the universal derivation `D` and another derivation still called `D`.